app/types/ts-proxies.d.ts

```diff
@@ -0,0 +1,14 @@
+// Import all optional peerDependencies types here to centralize all ts-ignore commands
+// These types will have `any` type at runtime until the appropriate package is installed
+// @ts-ignore
+import type * as ElectronBuilderProxy from "electron-builder";
+// @ts-ignore
+import type * as ElectronBuilderUtilProxy from "builder-util";
+// @ts-ignore
+import type * as ElectronPackagerProxy from "electron-packager";
+
+export {
+  ElectronBuilderProxy,
+  ElectronBuilderUtilProxy,
+  ElectronPackagerProxy,
+};
```

Comment:
```suggestion
```
I guess you previously were using `peerDependencies` when playing around, then decided to use `devDependencies` to avoid headaches. So, I guess we can get rid of this file completely, and use `import type` to import the types from the packages that are defined in `devDependencies` directly. That way we will not get errors since we will have those types at the build time, and the dependencies would not be bundled because they are `devDependencies` and we are not importing anything concrete because we are using `import type`. Please let me know if it's possible.

Reply:
Nope, this is intended
AFAIK this would result into TS errors in devland
If you try to import from a missing package, you'll get a TS error (hence the `@ts-ignore`s) unless you use `skipLibCheck`
If you import not-existing stuff and silence the error, you'll get an `any` for what you imported, which is what we want here

Into q/app everything works fine because `devDependencies` covers what we need, so this is a safeguard for devland which don't use these packages